js/common.js

```diff
@@ -1389,7 +1389,7 @@ function flashIconButton(button, button_classes, icon_classes, duration) {
 function uniqid(prefix = "", more_entropy = false) {
     const sec = Date.now() * 1000 + Math.random() * 1000;
     const id = sec.toString(16).replace(/\./g, "").padEnd(14, "0");
-    return `${prefix}${id}${more_entropy ? `.${Math.trunc(Math.random() * 100000000)}`:""}`;
```

Comment:
just a padStart(8, '0') was ok, no ?

Reply:
Yas, It would also work (although `padStart` is for strings so you would have to cast the value first).